parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/PLSQL.g4

```diff
@@ -300,7 +301,7 @@ dmlEventTrigger
     ;
 
 systemTrigger
-    : (BEFORE | AFTER | INSTEAD OF) (ddlEventTrigger (OR ddlEventTrigger)*) ON (PLUGGABLE? DATABASE) body
+    : (BEFORE | AFTER | INSTEAD OF) (ddlEventTrigger (OR ddlEventTrigger)* | databaseEventTrigger (OR databaseEventTrigger)*) ON ((PLUGGABLE? DATABASE) | (schemaName DOT_)? SCHEMA) body
```

Comment:
Hi, here is a small suggestion. We can rename `ddlEventTrigger` to `ddlEvent` and `databaseEventTrigger` to `databaseEvent`. We keep it consistent with the syntax diagram, so that maintenance can easily understand it in the same way as the syntax diagram, and other SQL statements can use this syntax quickly and easily.

Reply:
That's make sense. Thanks for your suggestion. Done.